src/gui/advancedsettings.cpp

```diff
@@ -436,6 +439,16 @@ void AdvancedSettings::loadAdvancedSettings()
     addRow(OS_MEMORY_PRIORITY, (tr("Process memory priority (Windows >= 8 only)")
         + ' ' + makeLink("https://docs.microsoft.com/en-us/windows/win32/api/processthreadsapi/ns-processthreadsapi-memory_priority_information", "(?)"))
         , &m_comboBoxOSMemoryPriority);
+
+    m_spinBoxMemoryWorkingSetLimit.setMinimum(0);
+    m_spinBoxMemoryWorkingSetLimit.setSingleStep(64);
```

Comment:
I would leave this out. Don't want to give users the impression that this value must be a multiple of 64.